### PR TITLE
HF correlations continued from the common repo

### PR DESCRIPTION
**Pull request by saganatt**
Dear all,

this is a continuation of https://github.com/AliceO2Group/Run3Analysisvalidation/pull/349.
The corresponding O2Physics PR is now: https://github.com/AliceO2Group/O2Physics/pull/1084

I am opening a new PR, as this is from a branch in our common repo, so Katarina and Gian can fix potential review comments and errors while I am offline.

### CODE REVIEW

**Review by vkucera — CHANGES_REQUESTED** [review]
- Remove the directory `originals` if not needed.
- Add basic documentation at the top of each .C macro:
  - What it does
  - How to use it
  - Who wrote it

File: FirstAnalysis/Correlations/extract2D.C [hunk 1]
```diff
@@ -0,0 +1,319 @@
+///////////////////////////////////////////////////////////////////////////
+//  Macro to produce 2D histograms of correlations vs. deltahi and deltaeta
```
Comment (vkucera) [inline FirstAnalysis/Correlations/extract2D.C:2]:
```suggestion
//  Macro to produce 2D histograms of correlations vs. delta phi and delta eta
```

**Review by vkucera — CHANGES_REQUESTED** [review]
Minor typo correction.

File: codeHF/config_tasks.sh [hunk 2]
```diff
@@ -85,6 +85,7 @@ DOO2_D0D0BAR_MCGEN=0     # hf-correlator-d0d0bar-mc-gen
 DOO2_DPLUSDMINUS_DATA=0  # hf-correlator-dplusdminus
 DOO2_DPLUSDMINUS_MCREC=0 # hf-correlator-dplusdminus-mc-rec
 DOO2_DPLUSDMINUS_MCGEN=0 # hf-correlator-dplusdminus-mc-gen
+DOO2_TASK_FLOW=0  # hf-task-correlation-d0
```
Comment (saganatt) [inline codeHF/config_tasks.sh:88]: leftover old name

File: codeHF/config_tasks.sh [hunk 3]
```diff
@@ -203,6 +204,15 @@ function AdjustJson {
     ReplaceString "\"processNoFT0\": \"true\"" "\"processNoFT0\": \"false\"" "$JSON" || ErrExit "Failed to edit $JSON."
   fi
 
+  # hf-task-correlation-d0
```
Comment (saganatt) [inline codeHF/config_tasks.sh:207]: leftover old name

**Review by ginnocen — APPROVED** [review]

File: codeHF/config_input.sh [hunk 4]
```diff
@@ -75,4 +75,9 @@ case $INPUT_CASE in
     INPUT_FILES="AO2D.root"
     ISINPUTO2=1
     ISMC=1;;
+  10)
+    INPUT_LABEL="Run 2, p-p 13.6 TeV, LHC17j (AO2D)"
```
Comment (vkucera) [inline codeHF/config_input.sh:79]:
```suggestion
    INPUT_LABEL="Run 2, p-p 13 TeV, LHC17j (AO2D)"
```

**Review by vkucera — APPROVED** [review]

**Pull request merged**